packages/core/backend/sirius-components-graphql-api/src/main/java/org/eclipse/sirius/components/graphql/api/TemporalScalarType.java

```diff
@@ -12,20 +12,22 @@
  *******************************************************************************/
 package org.eclipse.sirius.components.graphql.api;
 
+import java.time.temporal.Temporal;
+
 import graphql.schema.GraphQLScalarType;
 
 /**
- * Custom {@link GraphQLScalarType} for Instant.
+ * Custom {@link GraphQLScalarType} for {@link Temporal}.
```

Comment:
Stop using `{@link ...}` it just creates unnecessary imports and don't change this class, it is used by others.